client/src/pages/Home.js

```diff
@@ -1,20 +1,22 @@
 import { MapContainer, TileLayer, Marker, Popup } from "react-leaflet";
 import { useState, useEffect } from "react";
 import MarkerClusterGroup from "react-leaflet-markercluster";
+import CheckInText from "../components/CheckInText";
 
 export default function Home() {
-  const [playgrounds, setPlaygrounds] = useState([]);
+  const [playGroundData, setPlayGroundData] = useState([]);
   const [map, setMap] = useState(null);
+  const [status, setStatus] = useState();
```

Comment:
is status a german or english word? haha
could also be state, setState probably.